websites/N/Night.design/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BassKatze",
+		"id": "766213190332710913"
+	},
+	"contributors": [
+		{
+			"name": "Lena the fauli",
+			"id": "241278257335500811"
+		}
+	],
```

Comment:
A contributor must have contributed to at least 20% of the codebase, as this is a new presence there cannot be a contributor so far.

Reply:
But i have working on it as a dev but the owner of the project want him self as creator this is the reason why i have add me as contributors

Reply:
That's not how it works